pkg/aws/client.go

```diff
@@ -799,7 +793,7 @@ func (c *awsClient) GetAWSAccessKeys() (*AccessKey, error) {
 }
 
 func (c *awsClient) GetLocalAWSAccessKeys() (*AccessKey, error) {
-	creds, err := c.awsSession.Config.Credentials.Get()
+	creds, err := c.cfg.Credentials.Retrieve(context.TODO())
```

Comment:
NP
I couldn't find a major difference between `context.TODO()` and `context.Background()` in our case, as we don't set a deadline for the request.
I wonder if we should use `context.Background()`, just for the sake of consistency, not a must though.

Reply:
@oriAdler @den-rgb I think we should use `context.TODO()` consistently. From the Go API docs for `context.TODO()` :

```
TODO returns a non-nil, empty Context. Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).
```

I think that describes our situation. We don't have a parent context being passed down through, so we're not sure which context to use.

Reply:
@oriAdler @den-rgb FYI: https://issues.redhat.com/browse/OCM-5364